browser/js/cart/cart.controller.js

```diff
@@ -14,11 +16,10 @@ app.controller('CartController', function ($scope, cart, CartFactory) {
 	$scope.total = total();
 
 	$scope.update = function (item) {
-		if(item.quantity !== "") {
+		item.quantity = parseInt(item.quantity);
+		if(!isNaN(item.quantity)) {
```

Comment:
Maybe a try catch block would be best here so we can just display that error right?